apps/src/netsim/NetSimShard.js

```diff
@@ -128,3 +135,33 @@ NetSimShard.prototype.tick = function (clock) {
   this.messageTable.tick(clock);
   this.logTable.tick(clock);
 };
+
+/**
+ * The "panic button" option - clears all data on the shard, kicking all
+ * users out and starting over.
+ * @param {NodeStyleCallback} onComplete
+ */
+NetSimShard.prototype.resetEverything = function (onComplete) {
+  $.ajax({
+    url: '/v3/netsim/' + this.id + '/delete',
+    type: "post",
```

Comment:
I thought we were moving to a `DELETE /id` rather than a `POST /id/delete` model